bigquery_etl/cli/utils.py

```diff
@@ -33,13 +34,12 @@ def is_valid_file(ctx, param, value):
     return value
 
 
-def is_authenticated(project_id=None):
-    """Check if the user is authenticated to GCP and can access the project."""
-    client = bigquery.Client()
-
-    if project_id:
-        return client.project == project_id
-
+def is_authenticated():
+    """Check if the user is authenticated to GCP."""
+    try:
+        bigquery.Client(project="")
+    except DefaultCredentialsError:
+        return False
```

Comment:
this did not actually check if the user could access the project, and was never being called with project_id outside of tests.

additionally, the way it was being called implies that it should return `False` when not authenticated, rather than throwing an exception.